src/sentry/notifications/utils/participants.py

```diff
@@ -179,38 +181,58 @@ def get_reason(
 
 
 def get_participants_for_release(
-    projects: Iterable[Project], organization: Organization, user_ids: set[int]
+    projects: Iterable[Project], organization: Organization, commited_user_ids: set[int]
 ) -> ParticipantMap:
     # Collect all users with verified emails on a team in the related projects.
-    orm_users = User.objects.get_team_members_with_verified_email_for_projects(projects)
-    users = set(RpcActor.many_from_object(orm_users))
+    user_ids = (
+        OrganizationMember.objects.filter(
+            teams__in=Team.objects.filter(
+                id__in=ProjectTeam.objects.filter(project__in=projects).values_list(
+                    "team_id", flat=True
+                )
+            )
+        )
+        .distinct()
+        .values_list("user_id", flat=True)
```

Comment:
Should this be using joins instead of subqueries? 

```suggestion
        OrganizationMember.objects.filter(
            teams__projectteam__project__in=projects
        )
        .distinct()
        .values_list("user_id", flat=True)
```

I haven't run this, but it could work in theory.

Reply:
Yeah looks good to me, using.